server/channels/app/layer_generators/opentracing_layer.go.tmpl

```diff
@@ -140,4 +141,4 @@ func (a *{{.Name}}) Timezones() *timezones.Timezones {
 }
 func (a *{{.Name}}) SetServer(srv *app.Server) {
        a.srv = srv
-}
\ No newline at end of file
+}
```

Comment:
Is the generated file related to `server/channels/app/layer_generators/opentracing_layer.go.tmpl` already up to date?

Reply:
I've run `make generated` which I though it ran everything that should be run. Did I miss anything?